devito/passes/iet/misc.py

```diff
@@ -18,7 +19,7 @@ def avoid_denormals(iet):
     are normally flushed when using SSE-based instruction sets, except when
     compiling shared objects.
     """
-    if iet.is_ElementalFunction:
+    if iet.is_ElementalFunction or isinstance(archinfo.get_platform(), archinfo.Arm):
```

Comment:
The compilation passes (`iet_pass`) are independent of the underlying architecture.

If you don't want `avoid_denormals` on Arm (even though this appears to be a mere work around) the way to go is to create a suitable compilation pipeline for Arm, which is identical to Intel except for the lack of `avoid_denormals`.

There already exists one such Arm compilation pipeline: https://github.com/devitocodes/devito/blob/master/devito/core/__init__.py#L31 -- as you can see, named `ArmOperator`.

You should go to ArmOperator, intercept (override) `_specialize_iet` so as to drop `avoid_denormals`